src/main/java/uk/gov/nationalarchives/pdi/step/jena/model/JenaModelStep.java

```diff
@@ -303,7 +303,8 @@ private void closeAndThrow(final Model model, final String message) throws Kettl
     }
 
     private Object convertSqlValueToRdf(final Object sqlValue, @Nullable final RDFDatatype rdfDatatype) {
-        if (rdfDatatype == null || rdfDatatype.equals(XSDDatatype.XSDstring)) {
+        // Using reference equality because TypeMapper guarantees uniqueness in caller
+        if (rdfDatatype == null || rdfDatatype == XSDDatatype.XSDstring) {
```

Comment:
I actually think directly using reference equality is a bad idea in-case the API changes. At the moment calling .equals will use their API which ultimately does reference equality.